backends/cuda/ceed-cuda-operator.c

```diff
@@ -27,7 +27,7 @@ static int CeedOperatorDestroy_Cuda(CeedOperator op) {
     ierr = CeedVectorDestroy(&impl->evecs[i]); CeedChk(ierr);
   }
   ierr = CeedFree(impl->evecs); CeedChk(ierr);
-  ierr = CeedFree(impl->edata); CeedChk(ierr);
+  // ierr = CeedFree(impl->edata); CeedChk(ierr);
```

Comment:
I wonder why you're getting an error. We're going to want to free that so we don't leak.

Reply:
Sorry, that was lazy of me. Always the same error, I give the pointer I want to deallocate instead of the address of the pointer. I don't know why it didn't show up before though.